zstacklib/zstacklib/utils/jsonobject.py

```
@@ -190,28 +189,27 @@ def _dump(obj):
                 ret[key] = val
                 continue
 
-            nmap = _dump(val)
+            nmap = _dump(val, include_protected_attr=include_protected_attr)
             ret[key] = nmap
         elif isinstance(val, types.ListType):
```

Pylint (3.3.7)

[error] Module 'types' has no 'ListType' member

(E1101)